templates/definition/charger/tapo.yaml

```
@@ -0,0 +1,22 @@
+template: tapo
+products:
+- brand: TP-Link
+  description:
+    generic: TP-Link Tapo
```

```suggestion
    generic: Tapo
```
Brand und Description werden in `configure` und der Doku automatisch zusammengesetzt.